nav2_collision_monitor/src/polygon.cpp

```diff
@@ -168,6 +168,34 @@ bool Polygon::isShapeSet()
   return true;
 }
 
+bool Polygon::isUsingPolygonVelocitySelector()
+{
+  return !polygon_velocity_.empty();
+}
+
+void Polygon::updatePolygonByVelocity(const Velocity & cmd_vel_in)
+{
+  for (auto & polygon : polygon_velocity_) {
```

Comment:
`polygon` in this case will be of `PolygonVelocity` type. So, I'd like to suggest to rename it for better clarity to something like:
```
for (auto & polygon_velocity : polygons_velocity_) {
 ...
```

Does it make sense?

Reply:
Much better :100: